fixtures/american-dj/uv-eco-bar.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "UV Eco Bar",
+  "shortName": "uv-eco-bar",
+  "categories": ["Dimmer"],
```

Comment:
```suggestion
  "categories": ["Other"],
```

Reply:
You can open a PR against this PR's branch (`branch2024-09-03T11-06-45`).